app/models/winning_bidder_export.rb

```diff
@@ -29,7 +34,19 @@ def header_values
       '13PP Vendor Zip Code',
       '13QQ Vendor Country Code',
       '13RR Vendor Phone Number',
-      '13SS Vendor Fax Number'
+      '13SS Vendor Fax Number',
+      '2B Effective Date',
+      '2C Current Completion Date',
+      '3A Base And All Options Value',
+      '3B Base And Exercised Options Value',
+      '3C Action Obligation',
+      '6M Description of Requirement',
+      '6N Purchase Card as Payment Method',
+      '6R National Interest Action',
+      '9A DUNS Number',
+      '9B Contractor Name from Contract',
+      '10J Commercial Item Test Program',
+      '10M Solicitation Procedures'
```

Comment:
I am also against long classes, but when the majority of the class is just long arrays I tend think it's worth it to have the longer class rather than deal with additional indirection of throwing arrays in a new class. let's keep this one long for now and move stuff out of we feel like it gets too big/complicated